lua/autorun/server/sv_cfc_vote_init.lua

```diff
@@ -0,0 +1,18 @@
+local function addFiles( dir )
```

Comment:
It is redundant to name things in a server director sv_
same with shared and client